unsupported/juno/crates/cli/src/main.rs

```diff
@@ -277,8 +315,11 @@ fn run(opt: &Opt) -> anyhow::Result<TransformStatus> {
     drop(parsed);
     timer.mark("Cvt");
 
-    validate_tree(&mut ctx, &ast).with_context(|| input.display().to_string())?;
-    timer.mark("Validate AST");
+    // TODO throws for flow type nodes
+    if opt.dialect == Dialect::JavaScript {
+        validate_tree(&mut ctx, &ast).with_context(|| input.display().to_string())?;
+        timer.mark("Validate AST");
+    }
```

Comment:
`cargo run --package cli -- input.js --gen-js --strip-flow --dialect Flow`
throws 
```
flow.js:1:13: error: Unexpected NumberTypeAnnotation
flow.js: 1 AST validation errors
```
with this input
```js
function f(a: number) {
	return a;
}
```